map_dd : {
  map_00_lat : 35.0693571,
  map_00_lon : 128.5789111,
  map_00_alt : 53.0
  }

waypoint_List: {  # kyeong jun notion 
  waypoints:
  [ 
    #[35.069388456, 128.578979859833, 53.0], # 1
    [35.0695357, 128.5789378, 45.0], # 2
    [35.0694277, 128.5789276, 45.0], # 3
    [35.06960489, 128.5788552, 45.0], # 4
    [35.0693755, 128.5789723, 45.0], # 
    [35.069388456, 128.578979859833, 45.0]
    #[35.0693531, 128.5788206, 52.691],
    #[35.0693001, 128.578851, 52.691],
    #[35.0694106, 128.5787747, 53.002],
    #[35.0694728, 128.578313, 53.002],
    #[35.0694106, 128.5787747, 53.002]
     #[35.0694405, 128.5788903, 53.002],
     #[35.0694976, 128.5788951, 53.002],
     #[35.0695357, 128.5789378, 53.002],
     #[35.0693693, 128.5789487, 53.002]	
  ],
  }

goal_range : 1.0
# for `Waypoints` reached judgement

kp_servo : 0.45
ki_servo : 0.01 # max 0.05?
kd_servo : 0.1

kp_distance : 40

thruster_power : 1850

min_power : 1600

